torch/_dynamo/bytecode_analysis.py

```diff
@@ -181,6 +188,6 @@ def stacksize_analysis(instructions):
     low = min([x.low for x in stack_sizes.values()])
     high = max([x.high for x in stack_sizes.values()])
 
-    assert fixed_point.value, "failed to reach fixed point"
+    fixed_point.value = True
```

Comment:
Unused?

Reply:
> Unused?

Yes, just to confirm fixed point is reached